From b420f90b5dded55538a896639ab3c2473a5ca1ea Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@freescale.com>
Date: Wed, 19 Oct 2016 17:24:26 +0800
Subject: [PATCH 1227/5242] MLK-13369-2: ARM: dts: Add more parameters for gpr
 property of sound

commit  c446ead4489c9f1344f21d9e23facbf17f059f10 from
https://source.codeaurora.org/external/imx/linux-imx.git

The new parameter description is:
gpr = <gpr-node, register-offset, mask, value>;

Signed-off-by: Shengjiu Wang <shengjiu.wang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../arm/boot/dts/imx6ul-14x14-ddr3-arm2-wm8958.dts |    2 +-
 arch/arm/boot/dts/imx6ul-9x9-evk.dts               |    2 +-
 .../boot/dts/imx6ull-14x14-ddr3-arm2-cs42888.dts   |    2 +-
 .../boot/dts/imx6ull-14x14-ddr3-arm2-wm8958.dts    |    2 +-
 arch/arm/boot/dts/imx6ull-9x9-evk.dts              |    2 +-
 5 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2-wm8958.dts b/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2-wm8958.dts
index 3e53604..bdc5b90 100644
--- a/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2-wm8958.dts
+++ b/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2-wm8958.dts
@@ -47,7 +47,7 @@
 		cpu-dai = <&sai2>;
 		audio-codec = <&codec>;
 		codec-master;
-		gpr = <&gpr>;
+		gpr = <&gpr 4 0x100000 0x100000>;
 		hp-det-gpios = <&gpio5 0 1>;
 	};
 };
diff --git a/arch/arm/boot/dts/imx6ul-9x9-evk.dts b/arch/arm/boot/dts/imx6ul-9x9-evk.dts
index 11089a0..3d2c4ac 100644
--- a/arch/arm/boot/dts/imx6ul-9x9-evk.dts
+++ b/arch/arm/boot/dts/imx6ul-9x9-evk.dts
@@ -93,7 +93,7 @@
 		audio-codec = <&codec>;
 		asrc-controller = <&asrc>;
 		codec-master;
-		gpr = <&gpr>;
+		gpr = <&gpr 4 0x100000 0x100000>;
 		/*
                  * hp-det = <hp-det-pin hp-det-polarity>;
 		 * hp-det-pin: JD1 JD2  or JD3
diff --git a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-cs42888.dts b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-cs42888.dts
index b357282..cf14b6f 100644
--- a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-cs42888.dts
+++ b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-cs42888.dts
@@ -72,7 +72,7 @@
 		cpu-dai = <&sai2>;
 		audio-codec = <&codec_b>;
 		codec-master;
-		gpr = <&gpr>;
+		gpr = <&gpr 4 0x100000 0x100000>;
 		hp-det-gpios = <&gpio5 0 1>;
 	};
 };
diff --git a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-wm8958.dts b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-wm8958.dts
index b7c3afe..498281b 100644
--- a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-wm8958.dts
+++ b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-wm8958.dts
@@ -56,7 +56,7 @@
 		cpu-dai = <&sai2>;
 		audio-codec = <&codec_b>;
 		codec-master;
-		gpr = <&gpr>;
+		gpr = <&gpr 4 0x100000 0x100000>;
 		hp-det-gpios = <&gpio5 0 1>;
 	};
 
diff --git a/arch/arm/boot/dts/imx6ull-9x9-evk.dts b/arch/arm/boot/dts/imx6ull-9x9-evk.dts
index 590af08..47077b3 100644
--- a/arch/arm/boot/dts/imx6ull-9x9-evk.dts
+++ b/arch/arm/boot/dts/imx6ull-9x9-evk.dts
@@ -93,7 +93,7 @@
 		audio-codec = <&codec>;
 		asrc-controller = <&asrc>;
 		codec-master;
-		gpr = <&gpr>;
+		gpr = <&gpr 4 0x100000 0x100000>;
 		/*
                  * hp-det = <hp-det-pin hp-det-polarity>;
 		 * hp-det-pin: JD1 JD2  or JD3
-- 
1.7.9.5

